api/src/interpreter/interpreter.service.ts

```diff
@@ -1,16 +1,16 @@
 import { Injectable } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
-import { Repository } from 'typeorm';
-
 import { BookingDateEntity } from 'src/booking/entities/booking-date.entity';
+import { BookingPeriod } from 'src/booking/enums/booking-period.enum';
 import { SuccessResponse } from 'src/common/interface/response/success.interface';
+import { Repository } from 'typeorm';
```

Comment:
curios why moving this from line 3, we usually put npm package on the top.

Reply:
No idea. Possibly something vscode did automatically when I ran Shift + Option + F.